### PR TITLE
VolumeLables for unmanaged clusters

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/IBM/ibm-csi-common
 go 1.16
 
 require (
-	github.com/IBM/ibmcloud-volume-interface v1.0.1-beta4
+	github.com/IBM/ibmcloud-volume-interface v1.0.1-beta4.0.20220620123330-2ce06f9f3405
```
Comment (arahamad) [inline go.mod:6]: update with proper release

File: pkg/ibmcloudprovider/volume_provider.go [hunk 2]
```diff
@@ -78,6 +78,8 @@ func NewIBMCloudStorageProvider(configPath string, logger *zap.Logger) (*IBMClou
 			return nil, err
 		}
 		logger.Info("Fetched clusterInfo..")
+	} else {
```
Comment (arahamad) [inline pkg/ibmcloudprovider/volume_provider.go:81]: can we remove this else condition as csi driver main.go is generic to read the extra label

**Review by arahamad — CHANGES_REQUESTED** [review]

**Review by arahamad — APPROVED** [review]
lgtm

**Review by ambiknai — APPROVED** [review]
lgtm

**Pull request merged**